### ISSUE TITLE
Adding Learning Rate Schedules

### ISSUE THREAD

**Issue by SirRob1997** [facebookresearch/DomainBed#15]
Since optimizers are currently defined in `algorithms.py` and used in the `update` method, where would one add a learning rate schedule e.g. `torch.optim.lr_scheduler.ReduceLROnPlateau` to keep the design patterns?

**Comment by SirRob1997** [comment]
Sounds good, I'm on it
